sycl/include/CL/sycl/ordered_queue.hpp

```diff
@@ -118,6 +118,46 @@ class ordered_queue {
     });
   }
 
+  // single_task version with a kernel represented as a lambda.
```

Comment:
Does it mean `KernelFunc` can't be a functor or it's just a misleading comment?

Reply:
Misleading comment.

Reply:
Incidentally, these comments are copy/pasted from handler.hpp

Reply:
I guess we should update both files (handler.hpp and ordered_queue.hpp) then.
@romanovvlad, does it make sense to you?

Reply:
Yes, it does make sense.